mopidy_mpd/protocol/filter_expressions.py

```diff
@@ -0,0 +1,128 @@
+from mopidy_mpd import exceptions
+
+class peekable:
```

Comment:
The code in this file almost looks like it was copy+pasted from somewhere else, and definitely doesn't match the code style conventions of the codebase it's being inserted into.

Reply:
my implementation of `peekable` was modeled after the interface provided by more\_itertools, but stripped down for the use case here. the rest of the code is original, but written outside mopidy first.

for a more general take, i'm writing a 'real' comment outside the review box.